test/distributed_graph_generator_test_utils.h

```diff
@@ -0,0 +1,560 @@
+#pragma once
+
+#include <deque>
+#include <exception>
+#include <functional>
+#include <memory>
+#include <optional>
+#include <vector>
+
+#include <catch2/catch_message.hpp>
+#include <spdlog/fmt/fmt.h>
+
+#include "access_modes.h"
+#include "command_graph.h"
+#include "distributed_graph_generator.h"
+#include "print_graph.h"
+#include "task_manager.h"
+#include "types.h"
+
+#include "test_utils.h"
+
+using namespace celerity;
+using namespace celerity::detail;
+
+class dist_cdag_test_context;
```

Comment:
All of these definitions should be in `namespace test_utils`.

Reply:
Agreed!